Keycloak/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.2'
+
+services:
+
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: "keycloak"
+      POSTGRES_PASSWORD: ${ppassword}
+      POSTGRES_DB: "keycloak"
+    volumes:
+      - ./postgres:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
```

Comment:
Maybe we don't need to expose it since we'll only be calling keycloak service itself.